src/sidebar/services/groups.ts

```diff
@@ -507,8 +507,12 @@ export class GroupsService {
     const { signal } = this._focusedMembersController;
 
     this._store.startLoadingFocusedGroupMembers();
-    const members = await this._fetchAllMembers(groupId, signal);
-    this._store.loadFocusedGroupMembers(signal?.aborted ? null : members);
+    try {
+      const members = await this._fetchAllMembers(groupId, signal);
+      this._store.loadFocusedGroupMembers(signal?.aborted ? null : members);
+    } catch (e) {
+      this._store.focusedGroupMembersError(e);
+    }
```

Comment:
Since we load group members in chunks, this could probably be improved by saving the ones that succeeded, and if we attempt loading again, start from the last chunk that failed.

However, handling that properly is tricky, as over time the list of members is more likely to get outdated if we load one part at one point in time, and the rest later.

Additionally, it would complicate the logic quite a bit, so I think it's probably overkill.

With this approach, if loading any of the chunks fails, we discard everything and the next time we'll attempt again from scratch.